source/am_map.cpp

```diff
@@ -877,6 +877,36 @@ bool AM_Responder(const event_t *ev)
       if(ev->type != ev_keydown)
          return false;
 
+      // mousewheel zooming
+      if (ev->data1 == KEYD_MWHEELUP)
+      {
+         scale_mtof = scale_mtof * 1.3;
```

Comment:
Where's 1.3 come from?

Reply:
Sensitivity value for how much each scroll zooms in or out. I messed around until I found a decent value. I could add an option for it instead.

Reply:
Ah, figured as much, thanks. Honestly I think substituting it out for a constexpr is probably fine, both for the sake of clarity and also to make it easier if people want it as an option.